.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+<!--
+Thank you for contributing to the Dataverse Project!
+
+NOTE: Make sure you search the issue tracker to verify this bug has not been submitted already.
```

Comment:
I think lines 12-18 should be removed for now, since our issue tracker can be a bit hairy and this could be a big ask. I'd like to restore something like this in a future version of the template.

Reply:
Makes total sense.